example/ios/Podfile

```diff
@@ -4,13 +4,14 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 platform :ios, '10.0'
 
 target 'example' do
-  config = use_native_modules!
-
-  use_react_native!(:path => config["reactNativePath"])
 
   pod 'react-native-fbsdk', :path => '../node_modules/react-native-fbsdk'
 
   pod 'ReactNativeAdsFacebook', :path => '../node_modules/react-native-fbads'
```

Comment:
We can remove those as well (they both have .podspecs so they're automatically linked)